src/components/Navbar/SettingsDrawer/SettingsBody.tsx

```diff
@@ -1,19 +1,30 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 
 import QuranFontSection from './QuranFontSection';
 import ResetButton from './ResetButton';
 import ThemeSection from './ThemeSection';
 import TranslationSection from './TranslationSection';
 import WordByWordSection from './WordByWordSection';
 
-const SettingsBody = () => (
-  <>
-    <ThemeSection />
-    <QuranFontSection />
-    <WordByWordSection />
-    <TranslationSection />
-    <ResetButton />
-  </>
-);
+import { useOnboarding } from '@/components/Onboarding/OnboardingProvider';
+
+const SettingsBody = () => {
+  const { isActive, nextStep } = useOnboarding();
+
+  useEffect(() => {
+    if (isActive) setTimeout(() => nextStep(), 400); // wait for transition to finish (.4s)
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
We need to clear the timeout on un-mount. For reference: check `src/components/Verse/OverflowVerseActionsMenuBody.tsx`

Reply:
Also, where is the 0.4s coming from? CSS?